packages/waku/src/lib/plugins/vite-plugin-rsc-transform.ts

```diff
@@ -71,17 +71,26 @@ export ${name === 'default' ? name : `const ${name} =`} createServerReference('$
   }
 };
 
+export const createEmptySpan = (): swc.Span =>
+  ({
+    start: 0,
+    end: 0,
+  }) as swc.Span;
+
 const createIdentifier = (value: string): swc.Identifier => ({
   type: 'Identifier',
   value,
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-expect-error
+  ctxt: 0,
   optional: false,
-  span: { start: 0, end: 0, ctxt: 0 },
+  span: createEmptySpan(),
```

Comment:
Just to confirm, does it work if this is `span: { start: 0, end: 0 }`? Or, does TS complain?

Reply:
span: { start: 0, end: 0 }. swc type is broken

Reply:
How is it likely that swc will fix this type issue sometime soon-ish?

Reply:
/cc @kdy1 @magic-akari